bezier/src/cubic_bezier.rs

```diff
@@ -369,6 +455,22 @@ fn bounding_rect_for_cubic_bezier_segment() {
 }
 
 #[test]
+fn minimum_bounding_rect_for_cubic_bezier_segment() {
+    let a = CubicBezierSegment {
+        from: Point::new(0.0, 0.0),
+        ctrl1: Point::new(0.5, 2.0),
+        ctrl2: Point::new(1.5, -2.0),
+        to: Point::new(2.0, 0.0),
+    };
+    
+    let expected_minimum_bounding_rect = rect(0.0, -0.57735026, 2.0, 1.1547005);
```

Comment:
I would prefer to have a fuzzier test: something like checking that `rect(0.0, -0.6, 2.0, 1.2).contains_rect(&actual_minimum_bounding_rect)`. That way we should catch most regressions while not break the test every time we change the order of some operations and end up with slightly different rounding errors.